arch/arm64/common/uart_pl01x.c

```diff
@@ -118,15 +118,16 @@ uint8_t pl01x_read_byte(void)
 static void pl01x_handle_irq(void)
 {
 	irqc_disable_irq(UART_CON_IRQ);
-	pl01x_mask_irq(UART_CON_ID, UART_RXI);
+	pl01x_disable_irq(UART_CON_ID, UART_RXI);
```

Comment:
Shall keep the semantics of mask/unmask and re-implement mask/unmask API in the header file.